website/static/js/pages/project-dashboard-page.js

```diff
@@ -113,11 +113,14 @@ $(document).ready(function () {
         interactive: window.contextVars.currentUser.canEdit,
         maxChars: 128,
         onAddTag: function(tag){
-            var url = window.contextVars.node.urls.api + 'addtag/' + tag + '/';
+            var url = window.contextVars.node.urls.api + 'addtag/';
```

Comment:
`window.contextVars.node.urls.api` is saved to the variable `nodeApiUrl` in this file so you can use that instead: https://github.com/CenterForOpenScience/osf.io/blob/develop/website/static/js/pages/project-dashboard-page.js#L25-#L26